install/installer/cluster_dev.py

```diff
@@ -1060,13 +932,14 @@ def main():
     # elif cloud == 'DigitalOcean':
         # TODO
         # https://www.digitalocean.com/docs/apis-clis/doctl/how-to/install/
-        # cloud_login = get_do_login(cli.cloud_login)
-        # cloud_password = get_do_password(cli.cloud_password)
-        # cloud_token = get_do_token(cli.cloud_token)
+        # cloud_login = cli.cloud_login or get_do_login()
+        # cloud_password = cli.cloud_password or get_do_password()
+        # cloud_token = cli.cloud_token or ask_user('cloud_token')
 
         # create_cloud_user_and_req_permissions(cloud, cli.cloud_user, cloud_login, cloud_password)
 
-    cloud_provider = choose_cloud_provider(cli.cloud_provider, CLOUD_PROVIDERS[cloud])
+    cloud_provider = cli.cloud_provider or \
+        ask_user('choose_cloud_provider', choices=CLOUD_PROVIDERS[cloud])
```

Comment:
Escaping EOL is error-prone. Wrap with braces instead:
```suggestion
    cloud_provider = (
        cli.cloud_provider or
        ask_user('choose_cloud_provider', choices=CLOUD_PROVIDERS[cloud])
    )
```